src/icalendar/version.py

```diff
@@ -1,4 +1,20 @@
-"""Version file as a stable interface for the generated _version.py file."""
+"""Version in formation on icalendar.
+
+This module is the stable interface for the generated _version.py file.
+
+Version as a string:
+
+.. code-block:: pycon
+
+    >>> from icalendar import version
+
+Version as a tuple:
+
+.. code-block:: pycon
+
+    >>> from icalendar import version_tuple
```

Comment:
Examples would be clearer with expected output. Standard Python doc practice:

```suggestion
Version as a string:

.. code-block:: pycon

    >>> from icalendar import version
    >>> version  # doctest: +SKIP
    '7.0.0'

Version as a tuple:

.. code-block:: pycon

    >>> from icalendar import version_tuple
    >>> version_tuple  # doctest: +SKIP
    (7, 0, 0)
```

The `# doctest: +SKIP` directive is needed since the actual version changes with each release.

Reply:
Nice! I was looking for that one day. I really don't like changing the version number on each release!